javascript/code-challenges/challenges-06.test.js

```diff
@@ -40,7 +34,7 @@ Write a function that appends ' The end.' to a string, and returns the modified
 ------------------------------------------------------------------------------------------------ */
 
 const appendTheEnd = (str) => {
-  return str + `${" The end."}`;
+  return str + " The end.";
 };
```

Comment:
This is interesting. Looks like in the fist one you tried using template literals and it didn't work. Totally fair. What I find great about template literals it that it allows you to use variables directly in a string. Say you have to concat several variables into a string like so:
```js
str + ": The end of " + str2 + ". Will this sentence " + str3 + " make sense?";
```
That is a little tedius to read.
Now compare that with template literals
```js
`${str}: The end of ${str2}. Will this sentence ${str3} make sense?`
```
See how much easier it is to intuit the parts that str, str2, and str3 play in that sentence. It's just more natural to read.

Reply:
The reason yours didn't work the first time is that you attempted to use concatenation, and then put your string inside a template literal. You want to put your variable inside the template literal.
```js
return `${str} The end.`;
```